plugins/ltac/g_tactic.mlg

```diff
@@ -450,11 +450,11 @@ GRAMMAR EXTEND Gram
       | "("; nal=LIST1 name; ":"; c=lconstr; ")" -> { (nal,Default Glob_term.Explicit,c) }
     ] ]
   ;
-  fixdecl:
-    [ [ "("; id = ident; bl=LIST0 simple_binder; ann=fixannot;
+  ecase_fixdecl:
+    [ [ "("; id = ident; bl=LIST0 simple_binder; ann=ecase_fixannot;
         ":"; ty=lconstr; ")" -> { (loc, id, bl, ann, ty) } ] ]
   ;
-  fixannot:
+  ecase_fixannot:
```

Comment:
I don't think `ecase` makes sense here. I don't see where `fixdecl` is used elsewhere. So, I propose to keep `fixdecl` (or at worst to make it `fixtac_decl`).

As for `ecase_fixannot`, I would turn it e.g. into `structannot` or `fixtac_annot`.

Reply:
Maybe commenting on the wrong line?  `fixannot` is defined in both `g_constr` and `g_tactic`.  I'll change `ecase_fixdecl` back to `fixdecl`.

Reply:
and changed `ecase_fixannot`